app/views/guidance/before-you-appeal.html

```diff
@@ -62,10 +62,10 @@ <h2 class="heading-large">2. Appeal to the tribunal</h2>
               <li>reasons for your appeal, so the judge can understand your side of the argument</li>
             </ul>
               
-              <p>You will also need to complete an <a href="https://drive.google.com/open?id=0B-4PgEZDV9SBczhkbGNDanJEa28">authorisation form (PDF, 58Kb)</a></strong> if you want someone to represent you and they are not a legal professional.</p>
+              <p>You'll also need to download and fill in an <a href="https://drive.google.com/open?id=0B-4PgEZDV9SBczhkbGNDanJEa28">authorisation form </a></strong> if you want someone to represent you and they are not a practising solicitor or barrister.</p>
 
-            <p>
-            <input type="submit" class="button button-start" value="Start now">
+            <p><a href="https://tax-tribunals-datacapture-dev.dsd.io/">
```

Comment:
Do we really want to use the dev. URL here? You can't access it from outside the MoJ digital network.